example/main_tyxml.ml

```diff
@@ -28,16 +28,18 @@ let () =
   let tick () = Html.(h2 [ txt (Date.to_locale_time_string (Date.now ())) ]) in
   let svg_node =
     Svg.(
-      circle
-        ~a:
-          [ a_cx (50., None)
-          ; a_cy (50., None)
-          ; a_r (40., None)
-          ; a_stroke (`Color ("green", None))
-          ; a_stroke_width (4., None)
-          ; a_fill (`Color ("yellow", None))
-          ]
-        [])
+      svg
```

Comment:
That's the wrong `svg` function, so to speak. It's made for standalone SVG document. 

What you want is "SVG in HTML", aka `Html.svg`.